src/EntityRegistrationApp.tsx

```diff
@@ -60,10 +64,13 @@ const EntityRegistrationApp = (props: DataProps) => {
             .substring(1)
             .split("/")[0];
     }
-    setRegistryId(findRegistryIdentifierInPath());
 
+    const registryName = findRegistryIdentifierInPath();
+    
+    if(Boolean(registryName) && JSON.parse(registries).includes(registryName)){

```

Comment:
This assumes that the JSON of registries is always an array. Is this checked at any point, can we pass the array?